compose.static.yml

```
@@ -0,0 +1,141 @@
+services:
```

Compose files should specify scheme version, for example:
```suggestion
version: '3.7'
services:
```